endpoints-support.md

```diff
@@ -115,6 +115,8 @@
 |<sub>/rest/storage-volume-templates</sub>                                                | POST     | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/storage-volume-templates</sub>                                                | DELETE   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/storage-volume-templates</sub>                                                | PUT      | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Tasks**                                                                                                                                     |
```

Comment:
Task resource is not supported in Terraform SDK, please remove this entry

Reply:
But in endpoints file under vendor folder, there is an entry for Tasks. Do I need to remove there also.

Reply:
Vendor(Go-lang SDK) has a resource called Task, but in Terraform SDK Task resource doesn't exist. So please add only those endpoints which are supported in Terraform SDK.

Reply:
okay